public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/index.ts

```diff
@@ -0,0 +1,112 @@
+import type { Monaco, monacoTypes } from '@grafana/ui';
```

Comment:
This is Monaco specific-code, which glues the editor instance with the code in `completion.ts`.